css/style.scss

```diff
@@ -6,6 +6,7 @@
 @import "./scss/grid";
 @import "./scss/lists";
 @import "./scss/forms-input-select";
+@import "./scss/parallax-scroll";
```

Comment:
Consider putting the parallax-container class inside _grid.scss. It's debatable if it should be called "grid", maybe layout is a better name? 🤔